assets/lambda/cancelledTradeProxy.ts

```diff
@@ -0,0 +1,22 @@
+const log = require('lambda-log');
+import {handleEvent, ProxyFactory} from './lib/eventHandler';
+import {APIGatewayProxyEvent} from 'aws-lambda';
+import {CancelExecutedMgiTrade} from './lib/types';
+
+log.options.meta.logger = {
+  custom: true,
+};
+
+const factory: ProxyFactory<CancelExecutedMgiTrade> = (
+  detail: CancelExecutedMgiTrade
+) => {
+  return {
+    detail: detail,
+    getDestinationDetailType: () => `cancel-${detail.trading_model}-trade`,
+    source: 'CancelMgiExecutedTrade',
+  };
+};
+
+export const handler = (event: APIGatewayProxyEvent) => {
```

Comment:
for arrow function we can use shortcut notation without `return` keyword, let drop passing `log` as param.
```suggestion
export const handler = (event: APIGatewayProxyEvent) => handleEvent<CancelExecutedMgiTrade>(event, factory);
```